flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ProducerConfigConstants.java

```diff
@@ -20,14 +20,24 @@
 import org.apache.flink.streaming.connectors.kinesis.FlinkKinesisProducer;
 
 /**
+ * @deprecated
```

Comment:
@bowenli86 one thing to mention:
We should always try to have good Javadoc for why something is deprecated. Sorry but I overlooked this on my reviewing. I'll address this myself this time while merging.